src/components/search/SearchBlock.js

```diff
@@ -38,13 +47,31 @@ export default function SearchBlock({
       if (text?.length > 1) {
         const users = await usersService.search({
           login: text,
-          limit: 30,
+          limit: 5,
         });
-
         setSearchedUsers(users);
         setIsUserSearched(
           users.length ? null : "We couldn't find the specified user."
         );
+
+        if (isSearchOnlyUsers) {
+          return;
+        }
+
+        const conversations = await conversationService.search({
+          name: text,
+          limit: 10,
+        });
+        setSearchedChats(
+          conversations.reduce((chats, obj) => {
```

Comment:
This `reduce` does not have any sense. It just iterates over all found convs and add it to `chats ` array. What's the secret?

Reply:
Can't we just do `setSearchedChats(conversations)` ?

Reply:
From the server, we only have the chat `id` and the chat `name`, so here we fill the object with all the other parameters. In the future, this will also serve as a fillet for chats that are not in the user's list on the client (chat pagination)

Reply:
ok so I got it finally

it's just namings conflict. e.g. same naming

```
const conversations = useSelector(selectConversationsEntities);
```

```
const conversations = await conversationService.search({
          name: text,
          limit: 10,
        });
```

can we rename?